docs/topics/gradle.md

```diff
@@ -817,9 +817,10 @@ val customLauncher = service.launcherFor {
     it.languageVersion.set(JavaLanguageVersion.of(<MAJOR_JDK_VERSION>)) // "8"
 }
 project.tasks
-    .matching { it is UsesKotlinJavaToolchain && it.name == "compileKotlin" }
+    .withType<KotlinJvmCompile>()
```

Comment:
you can use `.withType(UsesKotlinJavaToolchain)`

Reply:
Oh that's right, I assumed it could not be of type `Task` because it was an interface, but `Task` is also an interface, and `UsesKotlinJavaToolchain` extends it.